AsyncSwift/Views/Profile/ProfileFriendDetailView.swift

```diff
@@ -0,0 +1,135 @@
+            friend: friend,
+            previous: previous
+        )
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            customDivider
+                .padding(.top, 10)
+            userDetail
+            Spacer()
+            linkButtons
+        }
+        .navigationTitle(observed.friend.name)
+        .navigationBarItems(trailing: trailingButton)
+        .fullScreenCover(isPresented: $observed.isShowingProfileSheet, content: {
+            SafariView(url: observed.friend.profileURL)
+                .ignoresSafeArea()
+        })
+        .fullScreenCover(isPresented: $observed.isShowingLinkedInSheet, content: {
+            SafariView(url: observed.friend.linkedInURL)
+                .ignoresSafeArea()
+        })
+        .alert("삭제", isPresented: $observed.isShowingConfirmAlert, actions: {
+            Button("삭제") { observed.isShowingConfirmAlert = false }
+            Button("확인") { observed.didConfirmDelete() }
+        }, message: {
+            Text("유저를 친구 목록에서 삭제하시겠습니까?")
+        })
+    }
+}
+
+private extension ProfileFriendDetailView {
+    var userDetail: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text(observed.friend.nickname)
+                .fontWeight(.semibold)
+                .font(.system(size: 20))
+            Text(observed.friend.role)
+                .fontWeight(.semibold)
+                .font(.system(size: 20))
+                .foregroundColor(.profileFontGray)
+                .padding(.bottom, 24)
+            Text(observed.friend.description)
+        }
+        .padding(.horizontal, 24)
+        .padding(.top, 28)
+    }
+
+    var linkButtons: some View {
+        VStack {
+            profileButton
+            linkedInButton
+                .padding(.bottom, 16)
+        }
+        .padding(.horizontal)
+    }
+
+    var profileButton: some View {
+        Button {
+            if observed.hasProfileURL() {
+                observed.isShowingProfileSheet = true
+            }
+        } label: {
+            Text("Profile URL")
+                .font(.headline)
+                .foregroundColor(.white)
+                .frame(minWidth: 0, maxWidth: .infinity, maxHeight: 68)
+                .background(observed.hasProfileURL() ? Color.seminarOrange : Color.inActiveButton)
+                .cornerRadius(15)
+        }
+    }
+
+    var linkedInButton: some View {
+        Button {
+            if observed.hasLinkedInURL() {
+                observed.isShowingLinkedInSheet = true
+            }
+        } label: {
+            Text("LinkedIn")
+                .font(.headline)
+                .foregroundColor(.white)
+                .frame(minWidth: 0, maxWidth: .infinity, maxHeight: 68)
+                .background(observed.hasLinkedInURL() ? Color.linkedInBlue : Color.inActiveButton)
+                .cornerRadius(15)
+        }
+    }
+
+    @ViewBuilder
+    var trailingButton: some View {
```

Comment:
```suggestion
    var navigationBarTrailingButton: some View {
```

어떤것에 대한 Trailing인지에 대해 이름이 명확해질 필요가 있어보여요 ~

Reply:
[refactor : Naming -> navigationBarTrailingButton](https://github.com/Async-Swift/AsyncSwift/pull/60/commits/1d955d2300b477ab80d8fcc4ba3fcb1ea4308072)

수정되었습니다 
감사합니다